scripts/tests/unit/fathomBridge/FathomBridge.test.js

```diff
@@ -0,0 +1,299 @@
+const chai = require('chai');
+const { expect } = chai
+const { solidity } = require("ethereum-waffle");
+chai.use(solidity);
+
+const { BigNumber, ethers } = require("ethers");
+
+const { WeiPerRad, WeiPerRay, WeiPerWad } = require("../../helper/unit");
+const { DeployerAddress, AliceAddress, BobAddress } = require("../../helper/address");
+const { getContract, createMock } = require("../../helper/contracts");
+const { loadFixture } = require("../../helper/fixtures");
+
+const { formatBytes32String } = ethers.utils
+const COLLATERAL_POOL_ID = formatBytes32String("WXDC")
+const ZeroAddress = "0x0000000000000000000000000000000000000000"
+
+const loadFixtureHandler = async () => {
```

Comment:
let's add some integration tests as well, it would be nice to test without mocking other protocol contracts

Reply:
Having integration tests is a great idea. I also wanted it. Unfortunately, it is not possible to deploy FathomBridge on ganache because its initialization depends on an external library that is already deployed on chain. And the external library gets triggered by off chain app.